src/AcceleratorTopoNull.hpp

```diff
@@ -46,8 +46,10 @@ namespace geopm
         public:
             AcceleratorTopoNull() = default ;
             virtual ~AcceleratorTopoNull() = default;
-            int num_accelerator(void) const override;
-            std::set<int> cpu_affinity_ideal(int accel_idx) const override;
+            virtual int num_accelerator(void) const override;
```

Comment:
don't think the virtual is required here, although not wrong.  Is the intention to have a class that inherits from AcceleratorTopoNull?